webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -169,16 +184,16 @@ class ManageManifestModal extends Component {
                       tooltipText={disabledReason}
                       tooltipPlacement="top"
                       title={__('Refresh')}
-                      disabled={actionInProgress || disableManifestActions}
+                      disabled={!this.manifestExist() || actionInProgress || disableManifestActions}
                     />
 
                     <TooltipButton
                       onClick={() => this.showDeleteManifestModal(true)}
                       tooltipId="delete-manifest-button-tooltip"
-                      tooltipText={__('This is disabled because a manifest task is in progress.')}
+                      tooltipText={__('This is disabled because a manifest task is in progress or no manifest exists')}
```

Comment:
How do you feel about changing this to call a function and get the actual reason that the button is disabled based on whether or not a manifest exists?